core/modules/replica/QhttpTestApp.cc

```diff
@@ -83,10 +88,11 @@ QhttpTestApp::Ptr QhttpTestApp::create(int argc, char* argv[]) {
 QhttpTestApp::QhttpTestApp(int argc, char* argv[])
     :   Application(
             argc, argv,
-            ::description,
-            false   /* injectDatabaseOptions */,
-            false   /* boostProtobufVersionCheck */,
-            false   /* enableServiceProvider */
+            description,
+            injectDatabaseOptions,
+            boostProtobufVersionCheck,
+            enableServiceProvider,
+            injectXrootdOptions
```

Comment:
None of those (older or newer) ways is ideal. In the older way, the comment would disappear after the code gets compiled and comments get eliminated, thus leaving the boolean literal in the code. An advantage of the "new" way is that these flags are named variables rather than literals "in the right position" in the method's call. I though, this would overweight inconveniences of the separate declaration and usage. That was the main motivation for the change.

Ideally, if C++ supported optional parameters in the Python-alike style, the code looked much better, and it wouldn't require compromizes. For example it could look like:
```
 Application(
    argc,
    argv,
    description,injectDatabaseOptions=true,
    boostProtobufVersionCheck=false,
     ...);
```
And here, I'd be on the same page with Andy S (regarding some usefulness of Python, not the language per se, but some syntactical "sugar" it has :)

Reply:
I agree, it would be nice if C++ supported named arguments.